libs/designsystem/src/lib/components/page/page.component.ts

```diff
@@ -259,6 +262,16 @@ export class PageComponent
     this.windowRef.nativeWindow.addEventListener(selectedTabClickEvent, () => {
       this.content.scrollToTop(KirbyAnimation.Duration.LONG);
     });
+
+    // Intercept back-button click events, defaulting to the built-in click-handler.
+    if (this.backButtonClicked.length == 0) {
```

Comment:
Are there a specific reason for using the equality - instead of the identity operator? 

Personally i prefer the more strict identity operator as it is more predictable how it will behave 😄 
```suggestion
    if (this.backButtonClicked.length === 0) {
```